components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -757,6 +766,695 @@ impl Interpolate for LengthOrNone {
         TransformList(Some(result))
     }
 
+    /// https://drafts.csswg.org/css-transforms/#Rotate3dDefined
+    fn rotate_to_matrix(x: f32, y: f32, z: f32, a: SpecifiedAngle) -> ComputedMatrix {
+        let rad = a.radians();
```

Comment:
can do a half_rad = a.radians()/2.0; instead as only half of the radian result is used everywhere else.